.yarnrc.yml

```diff
@@ -1,3 +1,5 @@
 nodeLinker: node-modules
 
 yarnPath: .yarn/releases/yarn-4.2.2.cjs
+
+enableGlobalCache: false
```

Comment:
This allows the build yarn cache to be moved from build platform to target platform (see Dockerfile)